python/astro_metadata_translator/translators/helpers.py

```diff
@@ -149,12 +149,16 @@ def altaz_from_degree_headers(
     altazpairs: tuple[tuple[str, str], ...],
     obstime: astropy.time.Time,
     is_zd: set[str] | None = None,
+    max_alt: float = 90.0,
```

Comment:
It might not be worth bothering with a max, although I know JCMT couldn't really get above 89 degrees.